src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerTest.java

```diff
@@ -152,32 +152,6 @@ public void tearDown() throws Exception {
                 .andExpect(status().isOk());
     }
 
-    @Test
```

Comment:
why remove these tests? maybe because tested in ws-commons?

Reply:
still the same issue, for testing controllers